recipes/mdspan/all/conanfile.py

```diff
@@ -26,12 +26,21 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
+            "Visual Studio": "15.1",
             "gcc": "5",
             "clang": "3.4",
             "apple-clang": "5.1"
         }
 
+    def _is_msvc(self, compiler_name):
```

Comment:
Please, consider `msvc` compiler. It should `1910` to be equivalent to 15.1.

Reply:
So please update accordingly. I guess 1911 should fits, as it's 15.3

Reply:
Also, please, update to [is_msvc](https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc)